### PR TITLE
Homepage: hero section copy update

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

Updated the copy of the hero section according to the new messaging

## Related Issue(s)

[#2525](https://github.com/FlowFuse/website/issues/2525)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/_data/site.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
         "d2fe": "https://boards.greenhouse.io/flowfuse/jobs/5185319004"
     },
     "messaging": {
-        "tagLine": "Elevate Node-RED with FlowFuse ",
-        "subtitle": "FlowFuse provides a complete platform for building, scaling and securing your production Node-RED applications. Accelerate digitalization of your industrial processes with ease.",
+        "tagLine": "The Fastest Way to Build and Operate Industrial Data Solutions",
+        "subtitle": "Connect, collect, transform, visualise, and interact with your Industrial Data in a single platform. We help manage, scale and secure your Node-RED solutions.",
         "title": "Node-RED for professionals",
```
Comment (Yndira-E) [inline src/_data/site.json:13]: @Hasmin-AC @ZJvandeWeg, what could we replace this with to better align it with the new messaging?

File: src/_data/site.json [hunk 2]
```diff
@@ -8,8 +8,8 @@
         "d2fe": "https://boards.greenhouse.io/flowfuse/jobs/5185319004"
     },
     "messaging": {
-        "tagLine": "Elevate Node-RED with FlowFuse ",
-        "subtitle": "FlowFuse provides a complete platform for building, scaling and securing your production Node-RED applications. Accelerate digitalization of your industrial processes with ease.",
+        "tagLine": "The Fastest Way to Build and Operate Industrial Data Solutions",
+        "subtitle": "Connect, collect, transform, visualise, and interact with your Industrial Data in a single platform. We help manage, scale and secure your Node-RED solutions.",
```
Comment (ZJvandeWeg) [inline src/_data/site.json:12]: Not sure about "we", maybe FlowFuse instead?

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by ZJvandeWeg** [comment]
Minor question from my side, but would be happy to merge now as changes are made in a minute

File: src/_data/site.json (hunk 1 above)
Comment (Hasmin-AC) [reply src/_data/site.json:13]: How about "Enterprise-ready Node-RED for Rapid Industrial Process Improvement"

File: src/_data/site.json (hunk 2 above)
Comment (Hasmin-AC) [reply src/_data/site.json:12]: yes we can change it to FlowFuse

Comment (Yndira-E) [reply src/_data/site.json:12]:
```suggestion
        "subtitle": "Connect, collect, transform, visualise, and interact with your Industrial Data in a single platform. FlowFuse helps manage, scale and secure your Node-RED solutions.",
```

File: src/_data/site.json (hunk 1 above)
Comment (Yndira-E) [reply src/_data/site.json:13]:
```suggestion
        "title": "Enterprise-ready Node-RED for Rapid Industrial Process Improvement",
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**